Include/cpython/interpreteridobject.h

```diff
@@ -0,0 +1,13 @@
+#ifndef Py_CPYTHON_INTERPRETERIDOBJECT_H
+#  error "this header file must not be included directly"
+#endif
+
+/* Interpreter ID Object */
```

Comment:
```c
#ifdef __cplusplus
extern "C" {
#endif
```
?

Reply:
Hmm, I could swear Victor had left that out of the other header files in Include/cpython.  Maybe I'm starting to lose it. :)

fixed